internal/controller/pkg/revision/runtime_test.go

```diff
@@ -125,30 +124,6 @@ func TestRuntimeManifestBuilderDeployment(t *testing.T) {
 				})),
 			},
 		},
-		"ProviderDeploymentWithImageOverride": {
```

Comment:
the logic that this test was covering now actually lives at a higher level - this logic is now covered by `TestGetProviderImage`